rfs_test/TEST_accountservice_schema.py

```diff
@@ -18,283 +18,419 @@
 
 import urllib3
 import json
+import string
+import random
 from openpyxl import load_workbook
 from openpyxl.styles import PatternFill
+import time
 from time import gmtime, strftime
 
 accSerData = None
+testAccountId = None
 
 # current spec followed for these assertions
 REDFISH_SPEC_VERSION = "Version 1.2.2"
 
+# Helper Functions
+
+def createDummyAccount(self):
+    authorization = 'on'
+    rq_headers = self.request_headers()
+    rq_body = {
+        'UserName': 'test_id',
+        'Password': 'test_pswd',
+        'RoleId': 'Administrator'
+    }
+    json_payload_get, headers, status = self.http_GET(self.sut_toplevel_uris['AccountService']['url'], rq_headers, authorization)
+
+    acc_collection = json_payload_get['Accounts']['@odata.id']
+
+    json_payload, headers, status = self.http_GET(self.sut_toplevel_uris['AccountService/' + acc_collection + '/test']['url'], rq_headers, authorization)
+    testAccountId = json_payload['Id']
+
+    return status
+
+def deleteDummyAccount(self):
+    authorization = 'on'
+    rq_headers = self.request_headers()
+    json_payload, headers, status = self.http_DELETE(self.sut_toplevel_uris['Accounts']['url'] + testAccountId, rq_headers, authorization)
+
+
+
+def failAuthorization(self):
+    authorization = 'on'
+    rq_headers = self.request_headers()
+    rq_body = {
+        'UserName': 'test_id',
+        'Password': 'wrong_pswd'
+    }
+
+    json_payload, headers, status = self.http_POST(self.sut_toplevel_uris['SessionService/Sessions']['url'], rq_headers, rq_body, authorization)
+
+    # The status code for authentication credentials included with this request being missing or invalid.
+    return status == 401
+    
+def isLocked(self):
```

Comment:
The "locked" check will be performed on the Manager Account itself, so you would need to use the response from the creation of dummy account, and then perform a GET on the URI from the Location header.

Reply:
Currently, I do not have access to a live version of Redfish, hence I do not know how the header for the POST request would look like.  I will apply this fix after I get access to a live Redfish API.

Reply:
Fixed based on your recommendation.